build/commands/lib/config.js

```diff
@@ -351,8 +351,8 @@ Config.prototype.buildArgs = function () {
   if (this.targetOS === 'ios') {
     args.target_os = 'ios'
     args.enable_dsyms = true
-    args.enable_stripping = args.enable_dsyms
-    args.use_xcode_clang = this.isOfficialBuild()
+    args.enable_stripping = !this.isDebug()
+    args.use_xcode_clang = false
```

Comment:
This is an intentional change, right? Does the 1.15/1.16 forks have this change? Just want to make sure we don't end up having issues submitting to Apple

Reply:
The change is intentional. This is what Chromium does for Xcode 12.0+. I explicitly changed this to `false` for `use_xcode_clang`. I have not tested submitting to Apple's servers but it did pass verification locally.